metadata-ingestion/tests/integration/tableau/test_tableau_ingest.py

```diff
@@ -69,8 +69,7 @@
     "projects": ["default", "Project 2", "Samples"],
     "extract_project_hierarchy": False,
     "page_size": 1000,
-    "workbook_page_size": 1000,
-    "embedded_datasource_field_upstream_page_size": 1000,
+    "workbook_page_size": None,
```

Comment:
`None` fallsback to `page_size`
so this is effectively the same as the previous `workbook_page_size: 1000`